src/client/views/components/todoItem.jsx

```diff
@@ -116,6 +118,17 @@ export default React.createClass({
         });
     },
 
+    makeElementFullyVisible: function () {
+        const { top: elementTop, bottom: elementBottom } = ReactDOM.findDOMNode(this).getBoundingClientRect();
```

Comment:
I think I suggested the spacing, but it doesn't match the rest of our app... I also noticed the line length is long here. How about:
```
const {top: itemTop, bottom: itemBottom} = ReactDOM.findDOMNode(this)
    .getBoundingClientRect(); 
```
or...
```
const {
    top: elementTop,
    bottom: elementBottom
} = ReactDOM.findDOMNode(this).getBoundingClientRect(); 
```

I'm not sure what looks best...